core/dns.go

```diff
@@ -14,6 +14,30 @@ import (
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/miekg/dns"
 )
 
+// Private CIDRs to ignore per RFC1918
+var (
+	// 10.0.0.0/8
+	privateNetworkA = net.IPNet{
```

Comment:
This A/B/C/D nomenclature is not standard.  It would be a little clearer to label these things by the mnemonics people already know; maybe something like `rfc1918_10`, `rfc1918_172_16`, `rfc1918_192_168`, `rfc4193`?  It's a little ugly, but I think it makes the semantic clearer.